client/src/components/Account/ProjectCard/ProjectCard.js

```diff
@@ -144,7 +144,13 @@ class ProjectCard extends React.Component {
                   })}
                   <h2>Reviews:</h2>
                   {this.state.reviews.map((rev) => {
-                    return <ReviewCard key={rev.id} review={rev} />;
+                    return (
+                      <ReviewCard
+                        key={rev.id}
```

Comment:
Are these cards more complicated than the ones we got on Home that require to be a separate component?

Reply:
Review Cards are more complicated, yes.

Reply:
In what sense? What makes them more complicated?

Reply:
Review cards have their own state and handle the functionality for reviews; allowing same user to edit/allowing others to like and dislike. The modal requirements are also fulfilled in the review card component. Featured cards are purely presentation focused. When a user clicks on the review from the project card, the modal should pop up w/ all the bells and whistles associated w/ reviews.